modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughSiteLevel.testcase

```diff
@@ -60,4 +66,69 @@ definition {
 		}
 	}
 
+	@description = "LPS-159844. When a walkthrough step is defined and a step is below the fold, user can see popover is."
+	@priority = "3"
+	test CanRenderWhenStepIsBelowFold {
+		property portal.acceptance = "true";
+
+		task ("Given walkthrough enabled site-level") {
+			Walkthrough.setupWalkthroughPage(
+				filename = "walkthrough_configuration_multiple_page.json",
+				siteName = "Site Name",
+				widgetList = "Blogs,JS Walkthrough Sample");
+		}
+
+		task ("Given a page width larger than the preview window.") {
+			SetWindowSize(value1 = "200,720");
```

Comment:
We can not add an image with the addWidgetToPublicLayout macro decided to decrease the screen size and add a blog widget, seems good?

Reply:
use frontend ckeditor sample portlet instead